poll/views.py

```diff
@@ -23,11 +23,25 @@ def home(request):
     up_movers = ranks.order_by('-ppv_diff')[0:5]
     down_movers = ranks.order_by('ppv_diff')[0:5]
 
+    dropped = []
+    prev_poll = Poll.objects.get(pk=(int(most_recent_poll.pk)-1))
+    if prev_poll.year == most_recent_poll.year: # don't display dropped teams for first poll of year
+        prev_top25 = PollCompare.objects.filter(poll=prev_poll).order_by('rank')[0:25]
+
+        teams = []
```

Comment:
I couldn't find a prettier way to do this. Is there a way to say "if team.team.pk not in top25.teams"? I'd have to compare the "rank.team" objects within top25 and prev_top25.

Reply:
Hmmm I kinda like that better.